go/vt/sidecardb/schema/vreplication/resharding_journal.sql

```diff
@@ -18,6 +18,6 @@ CREATE TABLE IF NOT EXISTS resharding_journal
 (
     `id`      bigint NOT NULL,
     `db_name` varbinary(255) DEFAULT NULL,
-    `val`     blob,
+    `val`     mediumblob,
```

Comment:
64KB -> 16MB, nice.

Reply:
I thought about using `longblob` but the db will most likely fall down if the gtid executed crosses 16MB!